DataFormats/L1TParticleFlow/src/layer1_emulator.cpp

```diff
@@ -34,14 +34,14 @@ bool l1ct::EmCaloObjEmu::write(std::fstream& to) const { return writeObj<EmCaloO
 
 bool l1ct::TkObjEmu::read(std::fstream& from) {
   src = nullptr;  // not persistent
-  return readObj<TkObj>(from, *this) && readVar(from, hwChi2) && readVar(from, hwStubs) && readVar(from, simPt) &&
-         readVar(from, simCaloEta) && readVar(from, simCaloPhi) && readVar(from, simVtxEta) &&
-         readVar(from, simVtxPhi) && readVar(from, simZ0) && readVar(from, simD0);
+  return readObj<TkObj>(from, *this) && readVar(from, hwChi2) && readVar(from, simPt) && readVar(from, simCaloEta) &&
+         readVar(from, simCaloPhi) && readVar(from, simVtxEta) && readVar(from, simVtxPhi) && readVar(from, simZ0) &&
+         readVar(from, simD0);
```

Comment:
Out of curiosity, why was `hwStubs` removed from the read and write?

Reply:
Because `hwStubs` is now a data member of the mother class (`TkObj`)